deployments/kustomize/backups/kustomization.yaml

```diff
@@ -0,0 +1,24 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+namespace: default
+resources:
+  - backup-db-cron.yaml
+  - backup-db-volumes.yaml
+commonLabels:
+  role: pmo-backups
+  app: gdx-agreements-tracker
+# Configuration for backup
+configMapGenerator:
+  - name: pmo-backup-config
+    literals:
+      - BACKUP_DIR=/backups/
+      - backup.conf=postgres=deployment:5432/postgres
+      - POSTGRES_DB=secret_postgres_db
+      - POSTGRES_USER=secret_postgres_user
+      - POSTGRES_REGULAR_USER=secret_postgres_regular_user
```

Comment:
Should we just remove this, as it is not going to be used.

Reply:
good point. I'll try to build/apply without it and see what happens